### DISCUSSION TITLE
Do the FastSam models work?

### DISCUSSION THREAD

**Discussion by torrinworx** [opengeos/segment-geospatial#156] [Q&A] [answered]
I'm trying to understand the scope of what Sam models work with the package, would I be able to use the [FastSam](https://github.com/CASIA-IVA-Lab/FastSAM#model-checkpoints) models (or other segmentation models) with segment-geospatial?

**Comment by giswqs** [comment, marked as answer]
Not yet, but we can add support for it. 
https://github.com/opengeos/segment-geospatial/issues/158
